src/main/kotlin/com/criptext/mail/scenes/composer/ComposerController.kt

```diff
@@ -610,7 +611,7 @@ class ComposerController(private val storage: KeyValueStorage,
         if(model.isReplyOrDraft || model.isSupport){
             scene.setFocusToComposer()
         } else {
-            scene.setFocusToTo()
+            if(model.to.isEmpty()) scene.setFocusToTo() else scene.setFocusToComposer()
```

Comment:
In what scenario the `to` is empty and is not a reply or draft or support?

Reply:
When the app recovers from a system termination.